components/webdriver_server/actions.rs

```diff
@@ -317,5 +358,148 @@ impl Handler {
                 .send(ConstellationMsg::WebDriverCommand(cmd_msg))
                 .unwrap();
         }
+
+        Ok(())
+    }
+
+    // https://w3c.github.io/webdriver/#dfn-dispatch-a-pointermove-action
+    fn dispatch_pointermove_action(
+        &mut self,
+        source_id: &str,
+        action: &PointerMoveAction,
+        tick_duration: u64,
+    ) -> Result<(), ErrorStatus> {
+        let tick_start = Instant::now();
+
+        let (start_x, start_y) = match self
+            .session
+            .as_ref()
+            .unwrap()
+            .input_state_table
+            .get(source_id)
+            .unwrap()
+        {
+            InputSourceState::Null => unreachable!(),
+            InputSourceState::Key(_) => unreachable!(),
+            InputSourceState::Pointer(pointer_input_state) => {
+                (pointer_input_state.x as i64, pointer_input_state.y as i64)
+            },
+        };
+
+        let x_offset = action.x.unwrap_or(0);
+        let y_offset = action.y.unwrap_or(0);
```

Comment:
When can the x and y values be unspecified?

Reply:
Steps 9 and 12 from https://w3c.github.io/webdriver/#dfn-process-a-pointer-move-action accept undefined values for `x` and `y` though their default value is not specified anywhere.